src/actions/wallet/L1/writeUnsafeDepositTransaction.ts

```diff
@@ -53,6 +53,16 @@ export type WriteUnsafeDepositTransactionParameters<
   _resolvedChain
 >
 
+/**
+ * Write directly to the Portal proxy to deposit ETH to L2
+ *
+ * @param {Address} to the address to send the transaction to
```

Comment:
I think we probably want to note that these are all wrapped as `args`, and that all WriteContractParameters can also added